File: MultipleVC/swift/MultipleViewControllers/SecondChildVC.m
//
//  SecondChildVC.m
//  MultipleViewControllers
//
//  Created by Коноплев Андрей on 09/11/2018.
//  Copyright © 2018 Коноплев Андрей. All rights reserved.
//

#import "SecondChildVC.h"

@interface SecondChildVC ()
@property (weak, nonatomic) IBOutlet UILabel *textLabel;

@end

@implementation SecondChildVC

- (void)viewDidLoad {
    [super viewDidLoad];
    _textLabel.text = @"Выберите страну или регион, чтобы контент и онлайн-товары соответствовали вашему местоположению.";
}



@end

File: MultipleVC/objc/MultiViewControllers/SecondChildVC.swift
//
//  SecondChildVC.swift
//  MultiViewControllers
//
//  Created by Коноплев Андрей on 09/11/2018.
//  Copyright © 2018 Коноплев Андрей. All rights reserved.
//

import UIKit

class SecondChildVC: UIViewController {

    @IBOutlet weak var tableView: UITableView!
    
    override func viewDidLoad() {
        super.viewDidLoad()

    }

}

extension SecondChildVC: UITableViewDelegate, UITableViewDataSource {
    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return 1000000
    }
    
    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let cell = UITableViewCell()
        cell.textLabel?.text = "cell num \(indexPath.row)"
        return cell
    }
    
    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
        return 40
    }
}
